include/seqan3/range/view/take.hpp

```diff
@@ -403,6 +406,18 @@ class view_take : public ranges::view_interface<view_take<urng_t, exactly, or_th
         ranges::copy(begin(), end(), ranges::back_inserter(ret));
         return ret;
     }
+
+    //!\overload
+    template <sequence_container_concept container_t>
+    operator container_t() const
+    //!\cond
+        requires std::CommonReference<reference_t<container_t>, reference> && const_iterable_concept<urng_t>
+    //!\endcond
+    {
+        container_t ret;
+        ranges::copy(cbegin(), cend(), ranges::back_inserter(ret));
```

Comment:
Don't we use the std::back_inserter now?

Reply:
Well, it was alle copy'n'paste, but your right, I can change it for the new ones. In general #498 has not been merged, yet.